src/documents/android/store/Store_IStoreAssets.html.md

```diff
@@ -110,7 +110,7 @@ public class ExampleAssets implements IStoreAssets{
   /** Virtual Categories **/
 
   public static VirtualCategory GENERAL_CATEGORY = new VirtualCategory (
-    "GENERAL", new ArrayList<String>(Arrays.asList(SHIELD_GOOD))
+    "GENERAL", new ArrayList<String>(Arrays.asList(new String[] { SHIELD_GOOD.getItemId() }))
```

Comment:
@vedi, you're talking about getter name or array declaration? If the first, there: https://github.com/soomla/android-store/blob/master/SoomlaAndroidStore/src/com/soomla/store/domain/VirtualItem.java#L144 defined the same method name.
If the second - I didn't see an error.